logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/slow/SlowSyncManager.kt

```diff
@@ -192,7 +192,7 @@ internal class SlowSyncManager(
          * Useful when a new step is added to Slow Sync, or when we fix some bug in Slow Sync,
          * and we'd like to get all users to take advantage of the fix.
          */
-        const val CURRENT_VERSION = 7
+        const val CURRENT_VERSION = 8
```

Comment:
Not sure about that, it won't hurt but will help the app to finally have full valid clients data.

Reply:
i don't think this is needed if we show the button on the screen in case the type is missing + afaik the list is refreshed when a user open that screen

Reply:
Ok, reverted that